.github/workflows/scorecard.yml

```diff
@@ -60,6 +60,6 @@ jobs:
 
       # Upload the results to GitHub's code scanning dashboard.
       - name: "Upload to code-scanning"
-        uses: github/codeql-action/upload-sarif@17573ee1cc1b9d061760f3a006fc4aac4f944fd5 # v2.2.4
+        uses: github/codeql-action/upload-sarif@00e563ead9f72a8461b24876bee2d0c2e8bd2ee8 # v2.21.5
```

Comment:
I verified this is the hash for the v2.21.5 release:
https://github.com/github/codeql-action/commit/00e563ead9f72a8461b24876bee2d0c2e8bd2ee8